server/src/utils/getConfig.ts

```diff
@@ -51,5 +99,17 @@ export const getClientConfig = (serverConfig: ServerConfigModel): ClientConfigMo
     waitingTitle: serverConfig.waitingTitle,
     waitingSubtitle: serverConfig.waitingSubtitle,
     logoUrl: serverConfig.logoUrl
-  };
+  } as ClientConfigModel;
+
+  if (serverConfig.postCall?.survey?.type) {
```

Comment:
At this point, the validations have already happened in `getServerConfig`. IMO we should just assign it directly:

```
postCall: serverConfig.postCall
```

Reply:
I am wondering if we just assign it directly and `serverConfig.postCall` is `undefined`, the config will be formulate as below:

![image](https://user-images.githubusercontent.com/94404633/184249170-e1be871d-f532-4a73-89ef-211d99791ca2.png)

Is this what we intended to?

Reply:
I think so. The client will render the default post call UI in that case.

Reply:
I thought since `postCall` is optional that `postCall` property shouldn't have to be in `config` if it the value doesn't exist.

Reply:
After we discussed offline, at this stage, it will be okay with `postCall: undefined`.